ui/src/frontend/viewer_page.ts

```diff
@@ -328,6 +328,11 @@ class TraceViewer implements m.ClassComponent<TraceViewerAttrs> {
                 }
                 globals.makeSelection(Actions.deselect({}));
               },
+              ondragstart: (e :DragEvent)=>{
```

Comment:
nit. And again, not caught by lint?
```suggestion
              ondragstart: (e: DragEvent)=>{
```